src/map/lua/luautils.cpp

```diff
@@ -1539,7 +1526,7 @@ namespace luautils
             else
             {
                 // Cache contentTag lookups in a map so that we don't re-hit the Lua file every time
-                contentEnabled = (GetSettingsVariable(contentVariable.c_str()) != 0);
+                contentEnabled = settings::get<bool>(fmt::format("main.{}", contentVariable)) != 0;
```

Comment:
No need for ` != 0;`, this is now a bool of its own and doesn't need to converted into one